src/components/Quote.vue

```diff
@@ -326,4 +331,7 @@ export default {
 	}
 }
 
+.file-preview {
```

Comment:
Was not sure about purpose of it. But if I got it right, it's not reducing the size of `<a>` tag
![image](https://github.com/nextcloud/spreed/assets/93392545/41dd712c-83c5-41ec-884f-56a9fce8b69d)